services/core-api/app/api/mines/permits/permit_amendment/resources/permit_amendment.py

```diff
@@ -132,6 +133,7 @@ def post(self, mine_guid, permit_guid, permit_amendment_guid=None):
             permit_amendment_type_code=permit_amendment_type_code
             if permit_amendment_type_code else 'AMD',
             description=data.get('description'),
+            security_total=data.get('security_total'),
```

Comment:
given this field is actually the delta of the security on the permit, we should change it's name, or at least leave a comment

Reply:
Added a comment to the modal